.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
```

Comment:
why this file?

Reply:
Not sure why, I merged the latest dev branch into the current in the mono-repo. Looks like this file was committed by you? Anyway we should remove it I guess?